spec/tomato_time_api_spec.rb

```diff
@@ -0,0 +1,14 @@
+require './spec/spec_helper'
+
+RSpec.describe 'TomatoTimeApi App' do
+  it "says hello world" do
+    get '/'
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('Hello world!')
+  end
+
+  it "displays a list of questions" do
+    get '/api/v1/questions'
+    expect(last_response).to be_ok
```

Comment:
Future test: assign `last_response.first` to a variable and check for its contents (aka, its attributes)